interfaces/IWear/include/Wearable/IWear/Actuators/IHaptic.h

```diff
@@ -27,6 +27,8 @@ class wearable::actuator::IHaptic : public wearable::actuator::IActuator
     inline static const std::string getPrefix();
 
     virtual bool setHapticCommand(double& value) const = 0;
+
+    virtual bool setHapticCommands(std::vector<double>& forceValue, std::vector<double>& vibrotactileValue) const = 0;
```

Comment:
Since it is a setter, it is convenient to use ``const``
```suggestion
    virtual bool setHapticCommands(const std::vector<double>& forceValue, const std::vector<double>& vibrotactileValue) const = 0;
```